cfpb/cfgov-refresh
# Migrate to terser from uglify for apps | Updates dependencies

## Changes

- Migrate to terser from uglify for apps scripts.
- Update to latest `@babel/core`, `@babel/preset-env`, `del`, `gulp`, `gulp-clean-css`, `lightbox2`, `webpack`, `lighthouse`, `protractor-retry`, and `snyk` in the main repo app.
- Update to latest `cf-core` in find-a-housing-counselor app.
- Update to latest `handlebars` in owning-a-home app.
- Update 

## Testing

1. Run `yarn`.
2. Run `gulp clean && gulp build`
3. http://localhost:8000/know-before-you-owe/compare/ lightboxes should work like production (click images).
4. http://localhost:8000/find-a-housing-counselor/ should work like production.
5. http://localhost:8000/owning-a-home/explore-rates/ and enter 600,000 for the loan amount. Text `Based on your loan amount, you may not be eligible for a regular (conforming) conventional loan.…` is a handlebars template and should appear.
6. `gulp audit:perf` should execute.
7. `yarn run test` should execute.
